pkg/rbac/authenticationMiddleware.go

```diff
@@ -30,20 +35,25 @@ func AuthenticateUser(next http.Handler) http.Handler {
 			return
 		}
 
-		authenticated, err := cache.IsValidToken(r.Context(), clientToken)
+		authenticated, err := Instcache.IsValidToken(r.Context(), clientToken)
```

Comment:
Can you change to `cacheInst` to follow naming convention?
- Using uppercase makes it public outside of the package, we want to keep the cache instance private and only access through public functions.

(I also hit problems because of the name collision, this helps to avoid confusion)